proto/qgb/genesis.proto

```diff
@@ -23,4 +23,4 @@ message GenesisState { Params params = 1; }
 message GravityNonces {
   // the nonce of the last generated validator set
   uint64 latest_valset_nonce = 1;
-}
\ No newline at end of file
+}
```

Comment:
Shouldn't we change `GravityNonces` -> `QgbNonces` for example? or even remove the prefix ?

Reply:
Not sure how that snuck in there, we don't need this struct at all.